### ISSUE TITLE
Audit #1 – General Overview when logged in – Text inputs' error state can't be detected programmatically                  

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4768]
Connected to Audit 1, see #4300
1.2 General Overview when logged in, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue

![CleanShot 2021-02-24 at 13 21 44](https://user-images.githubusercontent.com/505181/109000069-7d5e2200-76a3-11eb-8889-1bd4648230cc.png)

When a (text) inputs data is validated to be in an error state (like depicted), the error message must be associated with the invalid input. Also, the input should be marked as invalid itself.

# Remediation

Instead of:
```
<input aria-label="Ordnername" placeholder="Neuen Ordnernamen eingeben…" type="text" class="oc-text-input oc-text-input-danger"> 

<div class="oc-text-input-message"><!----> <span class="oc-text-input-danger">Neuer Ordner existiert bereits</span></div>
```

the Live DOM for inputs in an invalid state should be
```
<input aria-label="Ordnername" aria-invalid aria-describedby="errorMsg" placeholder="Neuen Ordnernamen eingeben…" type="text" class="oc-text-input oc-text-input-danger"> 

<div class="oc-text-input-message" id="errorMsg"><!----> <span class="oc-text-input-danger">Neuer Ordner existiert bereits</span></div>
```

Also, in case of an error, a live region (aria-live="assertive" or role=alert) should announce the error message.

Related BITV test step: 3.3.1a

**Comment by pascalwengerter** [comment]
Closing this since #5018 is merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
